pyt/usage.py

```diff
@@ -18,15 +18,6 @@
 )
 
 
-def valid_date(s):
```

Comment:
Can you remove https://github.com/python-security/pyt/blob/02461063688fe02226e627c00adfb2c707d89aa0/.coveragerc#L5 from `.coveragerc` as well?